### ISSUE TITLE
Issues with examples

### ISSUE THREAD

**Issue by michevnew** [MicrosoftDocs/OfficeDocs-Exchange#192]
In step 4, the New-AddressBookPolicy examples have an added space character in the name of the AL: " AL_FAB_Contacts"

In step 5, the examples use broken syntax. Here's a working example:

Get-Mailbox -ResultSize unlimited -Filter {CustomAttribute15 -eq 'FAB'} | Set-Mailbox -AddressBookPolicy 'ABP_FAB'

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c303bc4b-85f1-0946-287b-3dee7435c559
* Version Independent ID: 359332d1-e8e7-3320-7467-1edec873a18e
* Content: [Scenario Deploying address book policies in Exchange Server](https://docs.microsoft.com/en-us/Exchange/email-addresses-and-address-books/address-book-policies/abp-scenarios?view=exchserver-2019#detailed-deployment-steps-for-scenario-2-two-companies-sharing-a-ceo-in-one-exchange-organization)
* Content Source: [Exchange/ExchangeServer/email-addresses-and-address-books/address-book-policies/abp-scenarios.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/email-addresses-and-address-books/address-book-policies/abp-scenarios.md)
* Product: **exchange-server-it-pro**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@michevnew I removed the extra spaces from the code in Step 4 as you described.

For the code in Step 5, I did some of what you said and some of what was already there. Your proposed change would definitely work, but that `Get-Something | Set-Something` syntax can lead to throttling if the result size is too large, or even not that large (I've seen it myself).

Although the syntax is a bit more complex, it's always better to use the `Get-` cmdlet to store the results in a variable in one command, and then use that variable to make the change with the `Set` cmdlet in another command.

I changed the code blocks so they look like this:

```
$Fab = Get-Mailbox -ResultSize unlimited -Filter {CustomAttribute15 -eq 'FAB'}; $Fab | foreach {Set-Mailbox -Identity $_.Identity -AddressBookPolicy 'ABP_FAB'}
```

```
$Tail = Get-Mailbox -ResultSize unlimited -Filter {$_.CustomAttribute15 -eq 'TAIL'}; $Tail | foreach {Set-Mailbox -Identity $_.Identity -AddressBookPolicy 'ABP_TAIL'}
```

I used your syntax for the first commands, and I added the vital but missing `-Identity $_.Identity` piece to the second commands.

Thanks for the feedback.
